package-lock.json

```diff
@@ -32,7 +32,7 @@
         "@web3-onboard/trezor": "^2.3.2",
         "@web3-onboard/walletconnect": "^2.2.2",
         "@web3-onboard/web3auth": "^2.1.4",
-        "axios": "^0.26.0",
+        "axios": "^0.26.1",
```

Comment:
no package lock changes required in this PR

Reply:
I'll take a look at this.